composer.json

```diff
@@ -20,7 +20,7 @@
                 "version": "0.1.0",
                 "type": "drupal-library",
                 "dist": {
-                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-react/releases/download/0.1.0/dist.zip",
+                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-react/releases/download/release-release%2F1/dist.zip",
```

Comment:
Consider if we should refer to a tag here. `release/1` seems to be a moving target.

Whatever we do please update `version` accordingly.